content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1485,9 +1490,9 @@ Defines a [default value for a field](../../../concepts/components/prisma-schema
 
 #### Arguments
 
-| Name         | Required | Type                                      | Description |
-| ------------ | -------- | ----------------------------------------- | ----------- |
-| `expression` | **Yes**  | An expression (e.g. `5`, `true`, `now()`) |             |
+| Name         | Required | Type                                      | Description                                                              |
+| ------------ | -------- | ----------------------------------------- | ------------------------------------------------------------------------ |
+| `expression` | **Yes**  | An expression (e.g. `5`, `true`, `now()`) |                                                                          |
```

Comment:
This is missing `map` for now (which might be SQL Server only, so not sure how that works yet)